src/dgnss_management.c

```diff
@@ -609,7 +604,7 @@ s8 dgnss_low_latency_baseline(u8 num_sdiffs, sdiff_t *sdiffs,
   /* if we get here, we weren't able to get an IAR resolved baseline.
    * Check if we can get a float baseline. */
   s8 float_ret_code = _dgnss_low_latency_float_baseline(num_sdiffs, sdiffs,
-                                              ref_ecef, num_used, b);
+                                                        ref_ecef, num_used, b);
```

Comment:
I think it would ease merging to revert all changes to `dgnss_management.c` as its completely changed in my PR